src/biz/tugay/groovyship/cli/BoardCommandLinePrinter.groovy

```diff
@@ -0,0 +1,44 @@
+package biz.tugay.groovyship.cli
+
+
+import biz.tugay.groovyship.modal.Board
+import biz.tugay.groovyship.modal.Coordinate
+import biz.tugay.groovyship.service.ShipService
+
+class BoardCommandLinePrinter
+{
+  ShipService shipService = new ShipService()
+
+  void print(Board board) {
+    def noAttempt = ' '
+    def missedAttempt = '-'
+    def successfulAttempt = '□'
+    def sankShip = '◼'
+
+    print '   '
```

Comment:
When calling a method, parentheses is optional if the statement is a top-level statement like this one.

Reply:
in pyhton I especially like the if clause without parentheses:

```
if end_of_the_world == 666:
  pass
```